setup.py

```diff
@@ -53,17 +53,18 @@
 DOWNLOAD_URL = "https://github.com/pydicom/pydicom/archive/master.zip"
 LICENSE = "MIT"
 VERSION = __version__
-PACKAGE_DATA = {'pydicom': ['tests/test_files/*',
-                            'tests/charset_files/*']}
 REQUIRES = []
 DATA_FILES = []
 
 data_roots = ['pydicom/tests']
 
 for data_root in data_roots:
     for root, subfolder, files in os.walk(data_root):
-        files = [x for x in glob(root+'*') if not os.path.isdir(x)]
-        DATA_FILES.append((root, files))
+        files = [x.replace('pydicom/','') for x in glob(root+'/*')
+                if not os.path.isdir(x)]
+        DATA_FILES = DATA_FILES + files 
+
+PACKAGE_DATA = {'pydicom': DATA_FILES}
 
```

Comment:
Oh sorry - my bad! Misread the root path. But does that mean that the tests themselves are also included in the resources?